tests/pipeline/test_run.py

```diff
@@ -0,0 +1,62 @@
+import os
+import shutil
+from datetime import datetime
+
+import pandas as pd
+import yaml
+from chispa import assert_df_equality
+from mimesis.schema import Field
+from mimesis.schema import Schema
+
+from cishouseholds.pipeline.run import run_from_config
+from cishouseholds.pipeline.test_ETL import test_ETL
+
+_ = Field("en-gb", seed=11)
+
+
+def generate_test_csv(directory, file_date, records):
```

Comment:
What we really want to check here is whether a function that is named in a config can be called. The data here are over complicated to test this, which makes it unclear what is being tested.

The input data could be a csv with 1 row and 1 column, then the function that is called can just read and write this data to a new file.